src/server/lib/queryParams.ts

```diff
@@ -70,6 +71,7 @@ export const parseExpressQueryParams = (
 			componentEventParams || bodyParams.componentEventParams,
 		encryptedEmail,
 		error,
+		listName: listName || bodyParams.listName,
```

Comment:
Is there any possible scenario that the `listName` would come in a request body (req.body) and not query params?

As I don't think this parameter is set anywhere as a field on a form directly?

```suggestion
		listName: listName,
```

Reply:
No, it's only coming via query parameters. Thanks for this. This has been updated.